debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/transforms/UnwrapFromMongoDbEnvelopeTestIT.java

```diff
@@ -74,6 +75,51 @@ public void afterEach() {
         transformation.close();
     }
 
+    @Test
+    @FixFor("DBZ-563")
+    public void shouldDropTombstoneByDefault() throws InterruptedException {
```

Comment:
So it seems there should be a second test for actually keeping the tombstone? Or am I just missing it?

Reply:
The previous commit changes the existing test to turn off the tombstone drop as it was sort in the test path:
```
        final Map<String, String> transformationConfig = new HashMap<>();
        transformationConfig.put("drop.tombstones", "false");
        transformation.configure(transformationConfig);
```